artifact/README.md

```diff
@@ -0,0 +1,39 @@
+# Building & Using Artifact Container
+
+The `Dockerfile` in this directory creates a ready-to-use Docker container with compiled versions of Gobra.
+The full steps are as follows:
+1. Build the image:
+    ```
+    docker build -t gobraverifier/gobra-artifact:v1 .
+    ```
+   This downloads the specified versions of Gobra, Silver, Silicon, and Carbon into the image and compiles them to `/home/gobra/gobra.jar` and `/home/gobra/gobra-test.jar`.
+   `gobra.jar` just contains an executable version of Gobra whereas `gobra-test.jar` also includes the test classes. 
+2. Start the container:
+    - The following command starts an interactive shell in the container:
+        ```
+        docker run -it gobraverifier/gobra-artifact:v1
+        ```
+    - If you want to share a folder between your host and the container, use the following command that creates a new folder in the current directory on your host and mounts it to the container (it will be mounted at `/home/gobra/sync`):
```

Comment:
```suggestion
    - If you want to share a folder between your host and the container, use the following command that creates a new folder in the current directory on your host and mounts it to the container (it will be mounted at `/home/gobra/sync` inside the docker image):
```

Reply:
```suggestion
    - If you want to share a folder between your host and the container, use the following command that creates a new folder in the current directory on your host and mounts it to the container (it will be mounted at `/home/gobra/sync` inside the docker container):
```

Reply:
ok